### ISSUE TITLE
How to: Subject-Drive Generation, Face Swap (?)

### ISSUE THREAD

**Issue by ThomasAlbin** [Alpha-VLLM/Lumina-mGPT-2.0#6]
Hey Alpha-VLLM,

thanks for publishing this remarkable work! While the image generation works flawlessly, I am trying to create custom Python functions to call _inference_solver_. My code, e.g. to describe an image looks like this:

```
inference_solver = FlexARInferenceSolver(
    model_path="Alpha-VLLM/Lumina-mGPT-2.0",
    precision="bf16",
    target_size=768,
)

with torch.no_grad():
    generated = inference_solver.generate(
            images=[pic],
            qas=[[f"Depth map. <|image|>", None]],  # high-quality synthetic  superior
            max_gen_len=10240,
            temperature=1.0,
            logits_processor=inference_solver.create_logits_processor(cfg=4, image_top_k=4096),
        )
```
However, whether it is Depth Map (or derivatives of it), "Subject-driven generation", or a simple "Describe the image. <|image|>" I get in return:

> The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:None for open-end generation.

The output is a random set of strings. The input is an PIL.Image type.

I was wondering if you could provide a hint on how to e.g., conduct the subject-driven generation (like the example shown in the main page), or even a face swap with 2 images? I also tried to apply the logic from mgpt, but with no success (https://github.com/Alpha-VLLM/Lumina-mGPT/blob/main/lumina_mgpt/demos/demo_image2image.py).

Thanks a lot :)!

**Comment by nancywang1991-test** [comment]
I have the same question, thank you!

**Comment by synbol** [comment]
i2i will be open source in 1-2 weeks.

**Comment by ThomasAlbin** [comment]
@synbol thanks for your reply. I am looking forward to it :)!

**Comment by Should-AI-Lab** [comment]
> i2i will be open source in 1-2 weeks.

has i2i been opened?
